Source/CISQLite3/Private/SQLiteBlueprintFunctionLibrary.cpp

```diff
@@ -1,8 +1,8 @@
-#include "CISQLite3PrivatePCH.h"
-#include "Engine.h"
-#include "CString.h"
 
 #include "SQLiteBlueprintFunctionLibrary.h"
+#include "CISQLite3PrivatePCH.h"
+#include "Engine.h"
+#include "Misc/CString.h"

```

Comment:
I guess the PCH needs the BlueprintFunctionLibrary or why is this? 🤔

Reply:
I have no idea, I'm a complete noob in UE4 and C++ :laughing: 
I don't quite remember why, but this fixed some issues I had..

Reply:
Ah, found it. `CString.h` was moved to `Misc/CString.h` , and the order of imports was important, so they moved.